### PR TITLE
test unstructured.go module

### PR DESCRIPTION
**Pull request by phoracek**

### CODE REVIEW

**Comment by phoracek** [comment]
@SchSeba @qinqon

File: pkg/util/k8s/k8s_suite_test.go [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package k8s
```
[the remark this answers is not in the document]
Comment (phoracek) [reply pkg/util/k8s/k8s_suite_test.go]: Done

File: pkg/util/k8s/unstructured_test.go [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+			Expect(err).NotTo(HaveOccurred())
+
+			// Convert back to Pod object to check that no data was lost
+			unstructuredPodJSON, err := json.Marshal(unstructuredPod)
+			Expect(err).NotTo(HaveOccurred())
+			structuredPod := &apiv1.Pod{}
+			err = json.Unmarshal(unstructuredPodJSON, structuredPod)
+			Expect(err).NotTo(HaveOccurred())
+
+			Expect(structuredPod.Name).To(Equal("foo"))
```
[the remark this answers is not in the document]
Comment (phoracek) [reply pkg/util/k8s/unstructured_test.go]: Done

**Comment by phoracek** [comment]
ci test please

File: pkg/util/k8s/unstructured_test.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (qinqon) [reply pkg/util/k8s/unstructured_test.go]: fair enoough

**Pull request merged**